src/Features/colors/ColorHarmonies.tsx

```diff
@@ -14,14 +14,12 @@ import {
   triadic,
 } from "./harmonies";
 import { RenderShades } from "./RenderShades";
-import { useDebouncedValue } from "@mantine/hooks";
 import { notifications } from "@mantine/notifications";
+import { useColorRandomizer } from "./hooks";
 
 const ColorHarmonies = () => {
   const [color, setColor] = useState<string>("#61D2C3");
 
-  const [colorDeb] = useDebouncedValue(color, 300);
```

Comment:
Removed unnecessary rendering on the /color-util tab.

The elements were initially added on the color-utils since they were hidden by default. However, since they are no longer there, it is no longer necessary to debounce them on the /harmonies tab.

Reply:
fair. Can you also remove the unused imports and fix any other eslint errors in the same module? I saw some unused vars and imports in prev PR